src/modules/colorPicker/ColorPicker/dllmain.cpp

```diff
@@ -150,12 +232,46 @@ class ColorPicker : public PowertoyModuleIface
         if (m_enabled)
         {
             ResetEvent(send_telemetry_event);
+            ResetEvent(m_hInvokeEvent);
             TerminateProcess(m_hProcess, 1);
         }
 
         m_enabled = false;
     }
 
+    virtual bool on_hotkey(size_t hotkeyId) override
+    {
+        if (m_enabled)
```

Comment:
Let's add tracing here, since one of the goals of this PR is to fix several reports of problem with launching Color Picker.
Let's make sure we have a full tracing of this function to easily diagnostic what is going on in case of problems.

Reply:
Even if the module is disabled?

Reply:
No, if it's disabled let's not trace.